Makefile

```diff
@@ -325,3 +324,13 @@ install-gateway-api-local:
 .PHONY: uninstall-gateway-api
 uninstall-gateway-api:
 	kubectl delete -f $(GATEWAY_API_CRDS_LOCAL_PATH)
+
+### dev-env:			  Launch development environment
+.PHONY: dev-env
+dev-env: kind-up pack-image
+	./utils/install-apisix-with-helm.sh
+	kubectl set image deployment/apisix-ingress-controller ingress-controller=$(REGISTRY)/apache/apisix-ingress-controller:dev --namespace ingress-apisix
```

Comment:
change it to 

`kubectl set image deployment/apisix-ingress-controller ingress-controller=$(REGISTRY)/apisix-ingress-controller:dev --namespace ingress-apisix`

Reply:
![image](https://github.com/apache/apisix-ingress-controller/assets/43690894/07d8ad53-345b-4f5c-80ca-95ed632ae903)

Hi, anything else problems here?